src/types/organization.ts

```diff
@@ -83,3 +86,46 @@ export function fromPublicAdministrationToFoundAdministration(
     value => value
   );
 }
+
+export function fromOrganizationInstanceToOrganizationObject(
+  organizationInstance: OrganizationModel
+): Either<Errors, Organization> {
+  const legalRepresentative = {
+    email: organizationInstance.legalRepresentative.email,
+    family_name: organizationInstance.legalRepresentative.familyName,
+    fiscal_code: organizationInstance.legalRepresentative.fiscalCode,
+    given_name: organizationInstance.legalRepresentative.givenName,
+    phone_number: organizationInstance.legalRepresentative.phoneNumber,
+    role: organizationInstance.legalRepresentative.role
+  };
+  const users =
+    organizationInstance.users &&
+    organizationInstance.users.map(user => ({
+      email: user.email,
+      family_name: user.familyName,
+      fiscal_code: user.fiscalCode,
+      given_name: user.givenName,
+      role: user.role,
+      work_email: user.workEmail || undefined
+    }));
+  return Organization.decode({
+    fiscal_code: organizationInstance.fiscalCode,
+    ipa_code: organizationInstance.ipaCode,
+    legal_representative: legalRepresentative,
+    links: [
+      {
+        href: `/organizations/${organizationInstance.ipaCode}`,
+        rel: "self"
+      },
+      {
+        href: `/organizations/${organizationInstance.ipaCode}`,
+        rel: "edit"
+      }
+    ],
+    name: organizationInstance.name,
+    pec: organizationInstance.pec,
+    registration_status: organizationInstance.registrationStatus,
+    scope: organizationInstance.scope,
+    users
+  }) as Either<Errors, Organization>; // without casting the transpilation stucks
```

Comment:
remove this cast here, we'll try to resolve the issue together

Reply:
@gunzip, as you can see, the transpilation in the [last build](https://circleci.com/gh/teamdigitale/io-onboarding-pa-api/1111?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) gets stuck after removing this cast.